src/shared/components/ui/MonthRangePicker/Layout.test.js

```diff
@@ -1,84 +1,77 @@
 import React, { useState as useStateMock } from 'react'
-import { mount } from 'enzyme'
 import MonthRangePickerLayout from '@components/ui/MonthRangePicker'
-import MonthRangePicker from '@components/ui/MonthRangePicker/MonthRangePicker'
-import Button from '@components/ui/Button'
-import Spacer from '@components/ui/Spacer'
-
-const props = {
-  onChange: jest.fn(),
-  minDate: new Date(2021, 10, 11),
-  maxDate: new Date(2022, 10, 11),
-  i18n: {
-    apply: 'apply'
-  }
-}
+import { render } from '@jestutils'
+import { fireEvent } from '@testing-library/react'
+import '@testing-library/jest-dom'
 
 jest.mock('react', () => ({
   ...jest.requireActual('react'),
   useState: jest.fn()
 }))
 
-const selectedRangeMock = null
-
-const setRangeMock = jest.fn()
+describe('<MonthRangePickerLayout/> mount', () => {
+  const props = {
+    onChange: jest.fn(),
+    minDate: new Date(2021, 10, 11),
+    maxDate: new Date(2022, 10, 11),
+    i18n: {
+      apply: 'apply'
+    },
+    i18nHandler: jest.fn(translation => translation)
+  }
 
-const MonthRangePickerWrapper = <MonthRangePickerLayout {...props} />
+  const selectedRangeMock = null
 
-describe('<MonthRangePickerLayout/> mount', () => {
-  let wrapper
+  const setRangeMock = jest.fn()
 
   beforeEach(() => {
-    useStateMock.mockImplementation(() => [selectedRangeMock, setRangeMock])
-    wrapper = mount(MonthRangePickerWrapper)
-  })
-
-  afterEach(() => {
-    wrapper.unmount()
     jest.clearAllMocks()
+
+    useStateMock.mockImplementation(() => [selectedRangeMock, setRangeMock])
   })
 
   it('is mounted', () => {
-    expect(wrapper.exists()).toBe(true)
-  })
+    const { getByTestId } = render(<MonthRangePickerLayout {...props} />)
```

Comment:
Ogółem to Kent poleca, żeby używać tych funkcji getByTestId z obiektu screen, a nie destrukturyzować z render, bo wtedy mamy pewność, żeby nie musieć tu nic zmieniać, jak se będziemy wymieniać zapytania. Tutaj o tym pisze: https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#not-using-screen

Reply:
Spoko, szczerze nie zwracałem nawet na to uwagi, wiedziałem, że są dwie opcje, akurat ta mi się wydawała bardziej czytelna :D